coalib/parsing/DefaultArgParser.py

```diff
@@ -119,6 +120,26 @@ def default_arg_parser(formatter_class=None):
                                 metavar='BOOL',
                                 help='Write the logs as json to a file '
                                 'where filename is specified as argument.')
+    if parser_type == 'coala-html':
```

Comment:
Hm sure. That'd be better solution.  I'll try implementing it later this
evening.
On 17 Mar 2016 12:40, "Lasse Schuirmann" <notifications@github.com> wrote:

> In coalib/parsing/DefaultArgParser.py
> <https://github.com/coala-analyzer/coala/pull/1847#discussion_r56463497>:
>
> > @@ -119,6 +120,26 @@ def default_arg_parser(formatter_class=None):
> >                                  metavar='BOOL',
> >                                  help='Write the logs as json to a file '
> >                                  'where filename is specified as argument.')
> > +    if parser_type == 'coala-html':
>
> @tushar-rishav <https://github.com/tushar-rishav> he's right, if we want
> to use the arg parser cleanly we have to move coala-html into coala or add
> the arguments outside...
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala/pull/1847/files/5ee84784b42e6f34bb63f8ad833c0d99729b8d78#r56463497>
>